client/my-sites/email/email-providers-comparison/billing-interval-toggle/index.tsx

```diff
@@ -15,22 +17,40 @@ export const BillingIntervalToggle = ( {
 	onIntervalChange,
 }: BillingIntervalToggleProps ): ReactElement => {
 	const translate = useTranslate();
+	const [ payAnnuallyButtonRef, setPayAnnuallyButtonRef ] = useState< HTMLSpanElement | null >(
+		null
+	);
+
+	const isMonthlyPlan = intervalLength === IntervalLength.MONTHLY;
+	const isAnnuallyPlan = intervalLength === IntervalLength.ANNUALLY;
 
 	return (
 		<div className="billing-interval-toggle">
 			<SegmentedControl compact primary>
 				<SegmentedControl.Item
-					selected={ intervalLength === IntervalLength.MONTHLY }
+					selected={ isMonthlyPlan }
 					onClick={ () => onIntervalChange( IntervalLength.MONTHLY ) }
 				>
 					<span>{ translate( 'Pay monthly' ) }</span>
 				</SegmentedControl.Item>
 
 				<SegmentedControl.Item
-					selected={ intervalLength === IntervalLength.ANNUALLY }
+					selected={ isAnnuallyPlan }
 					onClick={ () => onIntervalChange( IntervalLength.ANNUALLY ) }
 				>
-					<span>{ translate( 'Pay annually' ) }</span>
+					<span ref={ setPayAnnuallyButtonRef }>{ translate( 'Pay annually' ) }</span>
+					{ [ 'right', 'bottom' ].map( ( pos ) => (
```

Comment:
Note we usually use [full words](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/javascript.md#naming-conventions) for variables:

```suggestion
					{ [ 'right', 'bottom' ].map( ( position ) => (
```